static/js/components/Main.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Provider } from 'react-redux';
+import { render } from 'react-dom';
+import { BrowserRouter, Link, Switch } from 'react-router-dom';
+import styled from 'styled-components';
+// Baselayer components
+import WebSocket from 'baselayer/components/WebSocket';
+import { Notifications } from 'baselayer/components/Notifications';
+// Message Handler
+import messageHandler from 'baselayer/MessageHandler';
+// Store
+import store from '../store';
+// Actions
+import hydrate from '../actions';
+import * as rotateLogoActions from '../ducks/logo';
+import PropsRoute from '../route';
+import {
+  NoMatchingRoute,
+  Responsive,
+  HomePage,
+  Source,
+  Groups,
+  Group,
+  Profile,
+  SourceList,
+  UserInfo,
+  Theme,
+  ProfileDropdown,
+  Logo,
+  Footer,
+} from '../components';
```

Comment:
ESLint doesn't like the `../components` import, but I find this a lot easier to read. You'll know exactly what imports are components vs. business logic.